target/linux/rockchip/image/armv8.mk

```diff
@@ -77,6 +77,14 @@ define Device/pine64_rockpro64
 endef
 TARGET_DEVICES += pine64_rockpro64
 
+define Device/radxa_cm3-io
+  DEVICE_VENDOR := Radxa
+  DEVICE_MODEL := CM3 IO
+  SOC := rk3566
+  DEVICE_DTS := rockchip/rk3566-radxa-cm3-io
```

Comment:
This board does not fit the default UBOOT_DEVICE_NAME value, so need to declare it here manually.
`UBOOT_DEVICE_NAME := radxa-cm3-io-rk3566`

Reply:
fixed